psscriptanalyzer/Initialize-Environment.ps1

```
@@ -0,0 +1,60 @@
+    foreach ($config in $configs) {
+        $content = cat $config.FullName
+        $parsed = $content | ConvertFrom-Json
+        Write-Host "loaded $($config.Name)"
+    }
+}
+
+function Process-Templates {
+    param([string]$TemplateDir)
+    cd $TemplateDir
+    ls *.tmpl | % {
+        $rendered = Invoke-Expression "Get-Content $($_.FullName) | Render-Template"
```

PSScriptAnalyzer (1.25.0)

[warning] Invoke-Expression is used. Please remove Invoke-Expression from script and find other options instead.

(PSAvoidUsingInvokeExpression)